app/models/collection_item.rb

```diff
@@ -187,10 +187,9 @@ def send_work_invitation
 
   after_update :notify_of_status_change
   def notify_of_status_change
-    if saved_change_to_unrevealed?
-      # making sure notify_recipients in the work model has not already notified 
-      # the user
-      if !work.new_recipients.blank?
+    if saved_change_to_unrevealed? && item.respond_to?(:new_recipients)
+      # making sure notify_recipients in the work model has not already notified
+      if item.new_recipients.present?
```

Comment:
This touches on the `item`/`work` divide on `CollectionItem`s, so I decided to double-check it.

In the old version of this code, this function doesn't do anything because `work` is fresh from the database, and so `new_recipients` is always blank (even when `item` has new recipients). In other words, this currently isn't capable of sending out any notifications under any circumstances, just because it uses `work` rather than `item`. So in theory, the switch from `work` to `item` here could result in duplicate notifications (from here and from wherever they're currently being sent out).

However, I don't think that's actually a problem here because this is an `after_update` hook, which means that with the current definition of `collection_names=`, this function can't be called while the work is being edited (which is what would cause `item.new_recipients` to be present). So at the moment, this function just plain doesn't do anything either way.

The change does prevent the error, though.

(Personally, I'd be inclined to delete the function entirely, but this also works.)